### PR TITLE
Fixes #212 Switch database for localhost from mysql-server back to original mysql

### LINKED ISSUE #212: Switch database for localhost from mysql-server back to original mysql
There was some things I noticed with the MySQL server switch we did on #186 

This is it switch back over to mysql and use `mysql:8-oracle`

-  It has a "warning" that it's built for Linux platforms and others aren't supported. They specifically mention Windows (and possibly OSX but it doesn't say that)
- I can't seem to get the root password to work and login with it still seeing if thats the same. I think it's because `MYSQL_ROOT_HOST` in mysql-server is more restrictive?
- I've been using `mysql:8-oracle` today and everything works plus the performance **seems** better

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Review by ssciolla — COMMENTED** [review]
I did some testing with this on my Intel machine, and I don't see any major issues with the change. If you think it's better for the M1 setup, I'll take your word for it.

However, I did notice that with the switch back, `python manage.py test` fails because the test database can't be properly provisioned. I noticed when this was changed earlier in #187  that we dropped the changes made in #161 to enable this, but it still worked, so I didn't worry too much about it at the time. Maybe we need to add them back if we're using the `mysql` image.

**Comment by jonespm** [comment]
Thanks! I've noticed this does works better on M1 and also on Windows. I know we don't have a lot of Windows users but @chrisrrowland and some students have worked on our projects in the past so we probably should have something consistent that works best for everyone.

**Comment by zqian** [comment]
This change works for my M1 too. I noticed the DBeaver connection broke recently for canvas_app_explorer_mysql, but that may be a different problem. 

I can still do `docker exec -it canvas_app_explorer_mysql mysql -u cae_user --password=cae_p` to get to mysql interface.

File: docker-compose.yml [hunk 1]
```diff
@@ -2,11 +2,10 @@ version: "3.9"
...
   mysql:
-    image: mysql/mysql-server:8.0
+    image: mysql:8-oracle
     restart: unless-stopped
     command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci', '--socket=/tmp/mysql.sock']
     environment:
-      - MYSQL_ROOT_HOST="0.0.0.0"
       - MYSQL_ROOT_PASSWORD=cae_root_pw
       - MYSQL_HOST=canvas_app_explorer_mysql
       - MYSQL_PORT=3306
```
[the remark this answers is not in the document]
Comment (jonespm) [reply docker-compose.yml]: I just changed it to the correct value, though I doubt we need to change it. I tested that` MYSQL_TCP_PORT` does work.

**Comment by jonespm** [comment]
I pushed the previous fix to run the init script. It can connect now but I don't have it configured.

Do we have the way to run this script and the environment variables documented anywhere?

**Review by ssciolla — APPROVED** [review]
LGTM!

**Pull request merged**